src/aggregate_score.rs

```diff
@@ -54,6 +71,38 @@ pub fn sample_aggregate_scores<const N_HAND: usize, R: Rng>(rng: &mut R, num_sam
     scores
 }
 
+pub fn parallel_sample_aggregate_scores<const N_HAND: usize, R: Rng>(
+    _rng: &mut R,
```

Comment:
We're passing a RNG here, but it is unused. We could just drop the argument, but I think that we'll want to keep this so that this function can be called in a repeatable way. Here is one way to do that:

- The caller explicitly sets the seed on `rng` before passing it in here.
- Inside of `parallel_sample_aggregate_scores` we use `rng` to compute a seed for the RNG used in each thread.

This would require changing the `&mut rand::thread_rng(),` line below to look more like:
https://github.com/MatthewPeterKelly/poker-stats/blob/ff1ade081eb2dc29eb68deddc404751bccc4e0f1/src/hand.rs#L77-L78

Once that is done, we can/should add a simple test to ensure that calling the `parallel_sample_aggregate_scores()` with the same input (including a RNG with the exact same state) will produce the exact same output.

Reply:
Thoughts on doing this?  I'm fine with just dropping the unused RNG argument for now and making a git issue to do the fancier thing with deterministic seeding.